sqld/src/http/stats.rs

```diff
@@ -7,13 +7,27 @@ use crate::stats::Stats;
 pub struct StatsResponse {
     pub rows_read_count: usize,
     pub rows_written_count: usize,
+    pub storage_bytes_used: u64,
+}
+
+impl std::convert::From<&Stats> for StatsResponse {
+    fn from(stats: &Stats) -> Self {
+        Self {
+            rows_read_count: stats.rows_read(),
+            rows_written_count: stats.rows_written(),
+            storage_bytes_used: stats.storage_bytes_used(),
+        }
+    }
+}
+
+impl std::convert::From<Stats> for StatsResponse {
```

Comment:
`From` is in the prelude

Reply:
also, that's a lot of code for saving an `&` 😄

Reply:
> also, that's a lot of code for saving an &
it is, but I also get really annoyed when other crates don't implement it and I need to figure it out myself every single time :innocent: this particular change is dedicated to future me